tests/test_resources/test_clusters/conftest.py

```diff
@@ -391,6 +391,119 @@ def local_docker_cluster_public_key(request):
         client.images.prune()
 
 
+def build_local_docker_cluster_with_telemetry(
```

Comment:
We probably need another round of merging the changes in this file with the changes from main:

https://github.com/run-house/runhouse/blob/d1993e3e8fce782fae5f1001bb7b5b5f32c07870/tests/test_resources/test_clusters/conftest.py#L335